localstack/cli/localstack.py

```diff
@@ -91,7 +91,7 @@ def cmd_status_services(format):
     url = config.get_edge_url()
 
     try:
-        health = requests.get(f"{url}/health", timeout=2)
+        health = requests.get(f"{url}/_localstack/health", timeout=2)
```

Comment:
Maybe we should add a fallback here (i.e. trying `/health` if `/_localstack/health` fails) in case they're using a newer CLI with an older localstack image version?

Reply:
The CLI has been introduced with `0.12.17`, and the `/_localstack/health` has been introduced with `0.13.0`.
I think it's safe to assume that the CLI is not used to managed a pre-v0.13 image.
The health check here is used in the Docker Health check, this is why I would not like to introduce a fallback which doubles the failure time...